spec/lib/ember-cli/deploy/redis_spec.rb

```diff
@@ -0,0 +1,108 @@
+require "spec_helper"
+require "ember-cli/deploy/redis"
+
+describe EmberCLI::Deploy::Redis do
+  describe "#append_to_head" do
+    it "injects the string into the <head> tag" do
+      provided_html = "<html><head></head></html>"
+      ember_cli_deploy = build_ember_cli_deploy(index_html: provided_html)
+
+      ember_cli_deploy.append_to_head("<script></script>")
+      index_html = ember_cli_deploy.html
+
+      expect(index_html).to eq("<html><head><script></script></head></html>")
```

Comment:
It would be nice to test with existing content in the `<head>` tag and make sure it's actually appended. The pull request to ember-cli-rails ended up prepending to head if called multiple times. Maybe consider a test like this.

```ruby
provided_html = "<html><head><title></title></head></html>"

ember_cli_deploy = build_ember_cli_deploy(index_html: provided_html)

ember_cli_deploy.append_to_head("<script></script>")
ember_cli_deploy.append_to_head("<meta></meta>")

expect(index_html).to eq("<html><head><title></title><script></script><meta></meta></head></html>")
```